### ISSUE TITLE
RuntimeError: "nll_loss_forward_reduce_cuda_kernel_2d_index" not implemented for 'Float'

### ISSUE THREAD

**Issue by hshany** [Oloren-AI/olorenchemengine#72]
Happens when fitting classification model with MLP.

To reproduce:
```
import olorenchemengine as oce
from tdc.single_pred import ADME


data = ADME(name = "cyp3a4_substrate_carbonmangels")
split = data.get_split()  
train = split['train']

params = {'BC_class_name': 'MLP',
 'args': [{'BC_class_name': 'DescriptastorusDescriptor',
   'args': ['rdkit2dnormalized'],
   'kwargs': {'log': True, 'scale': None}}],
 'kwargs': {'activation': 'leakyrelu',
  'batch_size': 16,
  'dropout': 0.1,
  'epochs': 100,
  'kernel_regularizer': 0.0001,
  'layer_dims': [2048, 512, 128],
  'lr': 0.0005}}
model = oce.create_BC(params)

model.fit(train["Drug"], train["Y"])
```

**Comment by davidzqhuang** [comment]
Thanks, will look into this

**Comment by davidzqhuang** [comment]
Should be fixed in commit: 4eb38c021f71a3b4bb6b4565f591caf8c9a7e156

to install this version please run:
`pip install --upgrade "olorenchemengine[full] @ git+https://github.com/Oloren-AI/olorenchemengine.git"`

please let me know if this doesn't work for you

**Comment by hshany** [comment]
It works now. Thanks.
